src/hazelweb/gui/CursorInspector.re

```diff
@@ -70,7 +70,8 @@ let exp_keyword_msg = (term, keyword, main_msg) =>
 
 let pat_ana_subsumed_msg =
     (expected_ty, got_ty, expecting_msg, consistency_msg) =>
-  if (HTyp.eq(expected_ty, got_ty) || HTyp.eq(got_ty, HTyp.Hole)) {
+  if (HTyp.eq(expected_ty, got_ty)
+      || HTyp.eq(got_ty, HTyp.Unknown(Internal))) {
```

Comment:
I think this should work for Unknown(_) not just Unknown(Internal) -- at least TypHole should work, and the other case won't actually happen I think